spacelift/data_policies.go

```diff
@@ -0,0 +1,74 @@
+package spacelift
+
+import (
+	"context"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+
+	"github.com/spacelift-io/terraform-provider-spacelift/spacelift/internal"
+	"github.com/spacelift-io/terraform-provider-spacelift/spacelift/internal/structs"
+)
+
+func dataPolicies() *schema.Resource {
+	return &schema.Resource{
+		Description: "" +
+			"`spacelift_policies` can find all policies that have certain labels," +
+			"and return their IDs. ",
+
+		ReadContext: dataPoliciesRead,
+
+		Schema: map[string]*schema.Schema{
+			"type": {
+				Type:        schema.TypeString,
+				Description: "type of the policy",
+				Required:    true,
+			},
+			"labels": {
+				Type:        schema.TypeSet,
+				Elem:        &schema.Schema{Type: schema.TypeString},
+				Description: "required labels to match",
+				Required:    true,
```

Comment:
I would not mark it as required. If someone doesn't want to match by label (which is a valid case IMO), we can allow them to not specify any labels.

Reply:
done, labels are optional